### DISCUSSION TITLE
Missing Lithuanian as Sky Culture language

### DISCUSSION THREAD

**Discussion by vaja0822** [Stellarium/stellarium#3400] [Translations]
I installed 23.2 version and now I can't find Lithuanian as Sky Culture language. I was able to use lithuanian in earlier versions.
In the log file I see the following message: "Couldn't load translations for language "lt_LT" in section "stellarium"
Empty translation file for language "lt_LT" in section "stellarium". Also, few more people checked - they also can't find lithuanian language.
What can be done to have lithuanian again as Sky Culture language?

**Comment by alex-w** [comment]
Lithuanian language was removed 11 months ago because Stellarium was translated into Lithuanian language lesser than 1% and long time has no any progress in translations. As far I can see you are one of two Lithuanian translators. I can add Lithuanian language and restore the translations from the backup copy, but where is guarantees than we'll see any progress for translation? The next release planned in two weeks (around 24 September) - do you will translation the Stellarium?

**Reply by vaja0822** [reply]
Yes, I've joined the translation team recently. I would be really grateful if you restore lithuanian language and I'll do as best as I can with all the translations in Lithuanian language.

**Reply by alex-w** [reply]
Done. Go ahead and good luck!

P.S. You are coordinator now

**Reply by gzotti** [reply]
Do we need lt_LT or just lt? (Same applies to requests for fr_FR, de_DE, it_IT,... when the base language has not been reasonably completed.)

**Reply by alex-w** [reply]
Just lt of course
